conf/broker.conf

```
@@ -1232,6 +1232,9 @@ exposeManagedCursorMetricsInPrometheus=false
 # Set it to 0 to disable timeout.
 metricsServletTimeoutMs=30000
 
+#Enable or disable the broker bundles metrics. default is false
+exposeBunlesMetricsInPrometheus
```

```suggestion
exposeBunlesMetricsInPrometheus=false
```
this shows the default value, right?